protocol/wallet.rei

```diff
@@ -1,14 +1,26 @@
 open Mirage_crypto_ec;
-open Helpers;
 
 [@deriving (ord, yojson)]
 type t;
+[@deriving (ord, yojson)]
+type pub_;
+
+let genesis_key: t;
+let genesis_pubkey: pub_;
+
+let wallet_of_privkey: Ed25519.priv => t;
+let wallet_to_privkey: t => Ed25519.priv;
+let pub_of_Ed25519pub: Ed25519.pub_ => pub_;
+let pub_to_Ed25519pub: pub_ => Ed25519.pub_;
```

Comment:
If they're alias and you will export the type like this, it's better to just export them with as `t = pri Ed25519.priv`

Reply:
Should I be using the `:>` syntax>